packages/database/test/exp/integration.test.ts

```diff
@@ -114,21 +115,20 @@ describe('Database@exp Tests', () => {
 
   // Tests to make sure onValue's data does not get mutated after calling get
   it('calls onValue only once after get request with a non-default query', async () => {
-    const db = getDatabase(defaultApp);
-    const { ref: testRef } = getUniqueRef(db);
+    const { readerRef } = getRWRefs(getDatabase(defaultApp));
     const queries = [
-      query(testRef, limitToFirst(1)),
-      query(testRef, startAt('child1')),
-      query(testRef, startAt('child2')),
-      query(testRef, limitToFirst(2))
+      query(readerRef, limitToFirst(1)),
+      query(readerRef, startAt('child1')),
+      query(readerRef, startAt('child2')),
+      query(readerRef, limitToFirst(2))
     ];
     await Promise.all(
       queries.map(async q => {
         const initial = [{ name: 'child1' }, { name: 'child2' }];
         const ec = EventAccumulatorFactory.waitsForExactCount(1);
-
-        await set(testRef, initial);
-        const unsubscribe = onValue(testRef, snapshot => {
+        const writerPath = getFreshRepo(readerRef._path);
```

Comment:
Why don't we just:
```
const { readerRef, writerRef } = getRWRefs(getDatabase(defaultApp));
```
above?

Reply:
I just wanted a fresh writerRef each time in this case, just to be safe.

Reply:
Ah I see.